docs/modules/ROOT/pages/db_services/alcor-transactions.adoc

```diff
@@ -0,0 +1,162 @@
+== Transactions in Alcor
+Prasad Kommoju <pkommoju@futurewei.com>
+
+v.01, 2021-05-05
+
+:toc: right
+
+== Overview
```

Comment:
@pkommoju Is this file included in other PR or merged to master already?

Reply:
It has been merged but I think I have made these changes after the merge.

Reply:
Actually the file alcor-transactions.adoc is under review in PR #611. Let us remove it from this PR and get it reviewed in PR #611

Reply:
I am having git push trouble. I had sent a query to github, got a reply. I am try it now.